.gitignore

```diff
@@ -1,3 +1,5 @@
-static/admin
+__pycache__/
```

Comment:
this matches any filenames from the root folder, that's why it's not finding `consumptions/_pycache_`